release/_releases_json.sh

```diff
@@ -48,7 +48,7 @@ function _process_product {
 	fi
 
 	for product_version in  $(echo -en "${directory_html}" | \
-		grep -E -o "(20[0-9]+\.q[0-9]\.[0-9]+|7\.[0-9]+\.[0-9]+[a-z0-9\.-]+)/" | \
+		grep -E -o "(20[0-9]+\.q[0-9]\.[0-9]+|7\.[0-9]+\.[0-9]+[a-tv-z0-9\.-]+)/" | \
```

Comment:
Hey @pgydev, is this commit excluding just the U releases starting from U113 as requested in the Expected Result of the https://liferay.atlassian.net/browse/LPD-22075?

Reply:
Besides, can you please visit the commit message and do the following:

- Start the commit message with capital letter, e.g. use `LPD-22075 Excluding` instead of `LPD-22075 excluding`
- Use the term `releases` instead of `versions`